contrib/hack/test_walkthrough.sh

```diff
@@ -135,6 +135,10 @@ retry -n 10 \
 
 echo 'Waiting on pods to come up...'
 
+wait_for_expected_output -x -e 'Pending' -n 10 \
```

Comment:
I feel like it is better to shoot for 'ready'; there's a chance you could miss something in a pending state and this wouldn't trigger.